src/Configuration/ConfigurationFactory.php

```diff
@@ -187,6 +187,8 @@ private function retrievePhpUnit(SchemaConfiguration $schema, string $configDir)
     }
 
     /**
+     * @param array<string, array<string>|bool> $schemaMutators
```

Comment:
I would not add this one or keep `array<string, mixed>`. The downstream type directly depends on the config we allow in `schema.json` and can get more complex than that without raising any warning here, resulting in a false assumption

Reply:
It's actually already incorrect in the case of `MBString`: which is `array<string, string[]|array<string, boolean>>` instead of `array<string>|bool`